src/scss/base/_utilities.scss

```diff
@@ -22,9 +22,10 @@
     &-left--#{$i} {
       margin-left: $horizontal-spacing;
     }
-
   }
-
+  &-top--half {
+    margin-top: $lineheight * 0.66;
```

Comment:
Just a comment, these magic numbers might start to become unmanageable,  at some point in the future it might be easier to manage them as variables